contracts/sol6/KyberNetwork.sol

```diff
@@ -986,10 +933,11 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils5, IKyberNetwork, Reentra
         }
 
         uint256 index;
+        bytes32[] memory rebateReserveIds = new bytes32[](tradeData.numEntitledRebateReserves);
 
         // token to ether
         index = populateRebateWalletList(
```

Comment:
need to update name.
maybe createRebateEntitledList

Reply:
Updated name